functions/go/Makefile

```diff
@@ -19,6 +19,7 @@ CURATED_GCR = gcr.io/kpt-fn
 # Edit this list to contain all go functions
 CURATED_FUNCTIONS := \
```

Comment:
Where's CONTRIB functions defined?

We should probably use a directory name convention to define curated vs contrib.

For example:

kpt-functions-catalog/contrib-functions/ts/sops/

OR

kpt-functions-catalog/functions/ts/contrib/sops/

Then we don't need this explicit list right?

Reply:
There are no contrib functions written in golang. So it's not defined here. But I have left a hint in the comment: 
https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/61dcfcb48a8339d6e24082932e32014810c14d91/functions/go/Makefile#L23
We have defined both CURATED and CONTRIB list for TS makefile:
https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/61dcfcb48a8339d6e24082932e32014810c14d91/functions/ts/Makefile#L26-L36

If using directory name convention, we don't need this list anymore. I'd the prefer the former.

Reply:
Not urgent, feel free to defer: but I think having `contrib` reflected in file hierarchy makes things more explicit.

Reply:
Filed https://github.com/GoogleContainerTools/kpt/issues/1480 to track it